docs/src/content/docs/fr/resources/community-content.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Contenu communautaire
+description: Découvrez les guides, articles et vidéos produits par la communauté pour vous aider à apprendre et à créer avec Starlight !
+---
+
+:::tip[Ajoutez le vôtre !]
+Avez-vous produit du contenu sur Starlight ?
+Ouvrez une PR ajoutant un lien à cette page !
+:::
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+## Articles et revues
+
+Voici une collection de publications et d'articles pour en apprendre plus sur Starlight et les expériences d'autres personnes :
+
+<CardGrid>
+	<LinkCard
+		href="https://devm.io/open-source/starlight-astro"
+		title="Génération de sites statiques avec Starlight"
+		description="« Aucune idée n'est trop grande ou trop petite lors de la conception de composants » — une entrevue avec Chris Swithinbank, responsable de Starlight"
+	/>
+	<LinkCard
+		href="https://frontendatscale.com/blog/hybrid-frontend-architecture/"
+		title="Une architecture frontend hybride avec Astro et Starlight"
+		description="Maxi Ferreira et Ben Holmes construisent un site de documentation avec Starlight, TinaCMS et une aire de test interactive avec authentification pour une API."
+	/>
+	<LinkCard
+		href="https://www.olets.dev/posts/comparing-docs-site-builders-vuepress-vs-starlight/"
+		title="Comparaison des générateurs de sites de documentation : VuePress contre Starlight"
+		description="Comment les deux frameworks se comparent-ils ?"
+	/>
+</CardGrid>
+
+## Recettes et guides
```

Comment:
Should "Recipes" be translated like the Astro Docs? My idea was to go there and grab the word but I decided against it as I personally don't really think it's conveying the idea or even well understood in isolation. This could totally be me so I decided to use the literal translation which I think makes way more sense in this context.